src/word_debt_bot/cogs/cmd_err_handler.py

```diff
@@ -0,0 +1,66 @@
+from discord.ext import commands
+
+import word_debt_bot.client as client
+import word_debt_bot.game as game
+
+
+class CmdErrHandler(commands.Cog, name="Command Error Handler"):
+    def __init__(self, bot: client.WordDebtBot, game: game.WordDebtGame):
+        self.bot = bot
+        self.game = game
+
+    @commands.Cog.listener()
+    async def on_command_error(self, ctx, err: commands.CommandError):
+        """The event triggered when an error is raised while invoking a command.
+        Parameters
+        ------------
+        ctx: commands.Context
+            The context used for command invocation.
+        error: commands.CommandError
+            The Exception raised.
+        """
+
+        # This prevents any commands with local handlers being handled here in on_command_error.
+        if hasattr(ctx.command, "on_error"):
+            return
+
+        if isinstance(err, commands.CommandNotFound):
```

Comment:
I'm inclined to just ignore this error, I find it annoying when bots complain about invalid commands when I wasn't trying to use them in the first place.

Reply:
Removed from local working version.